### ISSUE TITLE
Devices as columns

### ISSUE THREAD

**Issue by Bluscream** [custom-cards/flex-table-card#93]
I would like to have a device per row/colum and their same attributes as the opposite; kinda like this:

| Device   | Battery | Online | Triggered |   |
|----------|---------|--------|-----------|---|
| Device 1 | 50%     | Yes    | No        |   |
| Device 2 | 20%     | Yes    | Yes       |   |
| Device 3 | 5%      | No     | No        |   |

**Comment by ildar170975** [comment]
The only possible way is "One row = one entity", where each column may be filled:
- either with state & attributes' values;
- or with results of a JS code using data mentioned above.

i.e. the 2nd table cannot be made.

**Comment by Bluscream** [comment]
you're right, i wasnt sure if its even possible. But now that you told me it isn't, it turned into a featuee request ^^

**Comment by ildar170975** [comment]
> it turned into a featuee request ^^

I suggest you to change the issue's title then.
